consistent-sampling/src/main/java/io/opentelemetry/contrib/sampler/consistent/ConsistentParentBasedProbabilitySamplerProvider.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.sampler.consistent;
+
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
+import io.opentelemetry.sdk.autoconfigure.spi.traces.ConfigurableSamplerProvider;
+import io.opentelemetry.sdk.trace.samplers.Sampler;
+
+public final class ConsistentParentBasedProbabilitySamplerProvider
+    implements ConfigurableSamplerProvider {
+
+  @Override
+  public Sampler createSampler(ConfigProperties config) {
+    double samplingProbability = config.getDouble("otel.traces.sampler.arg", 1.0d);
```

Comment:
Would you want to force the `samplingProbability` into the interval [0.0, 1.0]? If out of the range, an exception will be thrown by the statement below.

Reply:
ConsistentProbabilityBasedSampler is already throwing an IllegalArgumentException if it's not within the range.

https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/consistent-sampling/src/main/java/io/opentelemetry/contrib/sampler/consistent/ConsistentProbabilityBasedSampler.java#L31-L33


```    
if (samplingProbability < 0.0 || samplingProbability > 1.0) {
  throw new IllegalArgumentException("Sampling probability must be in range [0.0, 1.0]!");
}
```

Reply:
Exactly my point. I just wondered if this is the behavior you want.

Reply:
Sorry, I misread your comment. 

Yes and I believe that was @oertl's intention as well.
TraceIdRatioBasedSampler in java sdk also has the same behaviour.
We could default to 1.0, but I think failing here would probably be more explicit.